data/systems/partial/02_local_stars.json

```diff
@@ -1,3 +1,409 @@
+    {"name": "Gamma Pavonis", "stars": ["STAR_F"], "sector": [-1, -2, -4], "pos": [0.216, 6.176, 4.672]},
+    {"name": "40 Eridani", "stars": ["STAR_K", "WHITE_DWARF", "STAR_M"], "sector": [1, -1, -1], "pos": [6.632, 0.8, 5.808], "comment": "Problem: the secondary and tertiary are co-orbital, and as a system orbit the primary"},
+    {"name": "Gliese 896", "stars": ["STAR_M", "STAR_M"], "sector": [-1, -3, 0], "pos": [5.656, 4.984, 6.952]},
+    {"name": "GJ 1123", "stars": ["STAR_M"], "sector": [0, 0, -4], "pos": [3.656, 4.232, 6.08]},
+    {"name": "Gliese 877", "stars": ["STAR_K"], "sector": [-1, -1, -4], "pos": [6.048, 1.216, 4.792]},
+    {"name": "NN 3533", "stars": ["STAR_K"], "sector": [3, 3, -2], "pos": [1.24, 2.84, 1.008]},
+    {"name": "GJ 1087", "stars": ["WHITE_DWARF"], "sector": [3, -1, 0], "pos": [1.952, 7.6, 2.44]},
+    {"name": "Chi Draconis", "stars": ["STAR_F"], "sector": [-1, -1, 3], "pos": [0.232, 7.288, 1.096]},
+    {"name": "Gliese 117", "stars": ["STAR_K"], "sector": [2, -4, -1], "pos": [6.576, 7.904, 0.52]},
+    {"name": "Gliese 116", "stars": ["STAR_K"], "sector": [3, -4, 3], "pos": [2.168, 3.968, 2.248]},
+    {"name": "107 Piscium", "stars": ["STAR_K"], "sector": [1, -3, 1], "pos": [1.88, 3.4, 0.44]},
+    {"name": "Gliese 299", "stars": ["STAR_M"], "sector": [2, 1, 0], "pos": [2.272, 3.848, 3.392]},
+    {"name": "AD Leonis", "stars": ["STAR_M"], "sector": [0, 1, 0], "pos": [6.376, 5.624, 5.44]},
+    {"name": "Groombridge 34", "stars": ["STAR_M", "STAR_M"], "sector": [0, -2, 1], "pos": [0.672, 7.656, 0.088]},
+    {"name": "Gliese 809", "stars": ["STAR_M"], "sector": [-1, -1, 2], "pos": [0.192, 0.624, 4.32]},
+    {"name": "Gliese 555", "stars": ["STAR_M"], "sector": [-2, 1, -1], "pos": [3.856, 7.224, 3.672]},
+    {"name": "Gliese 806", "stars": ["STAR_M"], "sector": [-3, -3, 3], "pos": [2.144, 4.824, 4.568]},
+    {"name": "33 G. Lib", "stars": ["STAR_K"], "sector": [-2, 1, -1], "pos": [3.464, 4.824, 0.968]},
+    {"name": "Gliese 412", "stars": ["STAR_M", "STAR_M"], "sector": [0, 1, 1], "pos": [2.688, 3.104, 2.856]},
+    {"name": "Gliese 361", "stars": ["STAR_M"], "sector": [2, 3, 1], "pos": [4.528, 5.64, 0.464]},
+    {"name": "Epsilon Indi", "stars": ["STAR_K"], "sector": [-1, -1, -2], "pos": [4.84, 2.344, 6.104]},
+    {"name": "Gliese 203", "stars": ["STAR_M"], "sector": [3, -1, 0], "pos": [3.68, 4.112, 4.752]},
+    {"name": "Xi Bo\u00f6tis", "stars": ["STAR_G", "STAR_K"], "sector": [-2, 1, 0], "pos": [1.96, 7.136, 7.152]},
+    {"name": "Gliese 205", "stars": ["STAR_M"], "sector": [2, -1, -1], "pos": [2.376, 5.696, 6.808]},
+    {"name": "NN 3976", "stars": ["STAR_M"], "sector": [-4, 1, 1], "pos": [3.216, 0.944, 4.456]},
+    {"name": "Alpha Centauri", "stars": ["STAR_G", "STAR_K"], "otherNames": ["Bungula", "Toliman", "\u03b1 Centauri", "Rigil Kentaurus", "Gliese 551", "Gliese 559", "FK5 538", "HIP 70890"], "sector": [-1, 0, -1], "pos": [6.624, 1.632, 4.16]},
```

Comment:
These should be using utf-8 codepoints, not ASCII-fied UTF-16 escapes (which is the default python `json.dump` behavior).

Reply:
fixed